app/components/chart-donut.js

```diff
@@ -43,6 +43,14 @@ export default Component.extend({
       return;
     }
 
+    if (!isIcon) {
+      svg.classed("svg-container", true)
```

Comment:
As I understand this code the class `.svg-container` gets added here, is that right?